product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -42,19 +42,18 @@ Users in the following roles can view the notifications:
 - Project owners/editors can view the project-level notifications.
 - Account owners can view their own account-level notifications. 
 
-Each notification indicates the level and/or project it belongs to for the user having multiple roles within BigAnimal.
+Each notification indicates the level and project it belongs to for the user having multiple roles in BigAnimal.
 
-Select the bell icon on the top of your BigAnimal portal to view the in-app notifications. On the bell icon, you can read the notification, mark it as unread and also archive the notification.
+Select the bell at the top of your BigAnimal portal to view the in-app notifications. By selecting the bell, you can read the notification, mark it as unread, and archive it.
 
-Check the inbox of your configured email addresses, to view the email notifications.
+To view the email notifications, check the inbox of your configured email addresses.
 
 ## Manage notifications
 
 To manage the notifications: 
 1. Log in to the BigAnimal portal.
-1. From the menu under your name in the top right panel, select **My Account**. 
-1. Select the **Notifications tab**. Notifications are grouped by organizations and projects available to you.
+1. From the menu under your name in the top-right panel, select **My Account**. 
+1. Select the **Notifications** tab. Notifications are grouped by organizations and projects available to you.
 1. Select any specific organization/project to manage the notifications.
-   - Enable/disable the notification for a particular event using the toggle button.
+   - Enable/disable the notification for a particular event using the toggle.
    - Select the **Email** and **Inbox** next to an event to enable/disable the email and in-app notifications for the event.
```

Comment:
I could quite envision this last step. Should the word "the" be deleted before **Email**?